File: sandbox/python/src/SignatureRequestRemindDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    signature_request_remind_request = models.SignatureRequestRemindRequest(
        email_address="john@example.com",
    )

    try:
        response = api.SignatureRequestApi(api_client).signature_request_remind(
            signature_request_id="fa5c8a0b0f492d768749333ad6fcc214c111e967",
            signature_request_remind_request=signature_request_remind_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling SignatureRequest#signature_request_remind: %s\n" % e)

File: sandbox/node/src/SignatureRequestRemindDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.SignatureRequestApi();

const signatureRequestRemindRequest = new models.SignatureRequestRemindRequest();
signatureRequestRemindRequest.emailAddress = "john@example.com";

const signatureRequestId = "fa5c8a0b0f492d768749333ad6fcc214c111e967";

apiCaller.signatureRequestRemind(
    signatureRequestId,
    signatureRequestRemindRequest,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling SignatureRequest#signatureRequestRemind:");
  console.log(error.body);
});
